test/builder/eventlistener.go

```diff
@@ -313,3 +320,37 @@ func EventListenerCELOverlay(key, expression string) EventInterceptorOp {
 		}
 	}
 }
+
+// EventListenerResources set specified resources to the EventListener.
```

Comment:
In pipelines, we've been trying to get rid of test builders and use structs as much as possible. For our reconciler tests too, we've started using single make(EL/Deployment/Service) functions. I think its ok to keep for now, but later we might end up removing builders just like in Pipeline.